### PR TITLE
package.json

### PR DESCRIPTION
**Pull request by michaelcullum**
A package.json based on the new spec.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "phpbb-blog/phpbb-blog",
```
Comment (imkingdavid) [inline package.json]: should be "naderman/example"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "phpbb-blog/phpbb-blog",
+    "type": "phpbb3-extension",
+    "description": "This will add an AI with mind reading capabilities to your board who will post your thoughts to a blog",
```
Comment (imkingdavid) [inline package.json]: copy/paste fail much? :P

File: package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "phpbb-blog/phpbb-blog",
+    "type": "phpbb3-extension",
+    "description": "This will add an AI with mind reading capabilities to your board who will post your thoughts to a blog",
+    "homepage": "http://yourdomain.com/",
```
Comment (imkingdavid) [inline package.json]: perhaps a link to this github repo?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            "name": "naderman",
+			"email": "naderman@naderman.de",
+			"homepage": "http://naderman.de",
+            "type": "Lead Developer"
+        },  
+        {
+            "name": "imkingdavid",
+			"email": "imkingdavid@gmail.com",
+			"homepage": "http://thedavidking.com",
+            "type": "Contributions"
```
Comment (imkingdavid) [inline package.json]: Contribut_or_ (same for igorw and you)

**Comment by imkingdavid** [comment]
This will need to wait until #1 is merged. Michael didn't want to base off of my pull so that you wouldn't have to click merge twice, sorry.

File: package.json (hunk 3 above)
Comment (michaelcullum) [inline package.json]: Sorry, I updated these, then pasted something in and accidently overwrote everything but I didn't notice.

**Comment by igorw** [comment]
Indentation is messed up.

**Comment by naderman** [comment]
Formatting is still somewhat messed up. There is no reason to have { on a separate line in json, and we've never had spaces between closing brackets and there should be a newline at the end of file.

**Comment by michaelcullum** [comment]
Better?

**Comment by naderman** [comment]
Only thing I'm not sure about is, why are all the author types "Current X"? Isn't that kind of implied?

**Comment by michaelcullum** [comment]
DavidIQ and imkingdavid asked for it. Its all in the new spec. http://area51.phpbb.com/phpBB/viewtopic.php?f=84&t=42567

Current and Previous Prefix what they do. He wanted to be able to show position and previous or current. Similar to how its done in MODX except combined into one.

Author username, the extras and author type are the only things custom for phpBB. The rest is all composer based.

**Comment by naderman** [comment]
I guess I'll ask there then. I'm fine with a previous prefix, but current seems superfluous.

**Pull request merged**